base/applications/utilman/lang/zh-CN.rc

```diff
@@ -0,0 +1,58 @@
+/*
+ * PROJECT:         ReactOS Utility Manager (Accessibility)
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         Simplified Chinese translation resource
+ * COPYRIGHT:       Copyright 2020 robsean (robsean@126.com)
+ */
+ 
+/* Chinese text is encoded in UTF-8 */
+#pragma code_page(65001)
```

Comment:
This will fail as your translation file is not included in the main resource file of the program. See https://github.com/reactos/reactos/blob/master/base/applications/utilman/utilman.rc

The macro definition must be added in the alphabetical order. And as we're at it, this line is useless so remove it.

Reply:
ok，I saw that other Chinese documents had this line ,so I added it